cornice/util.py

```diff
@@ -128,7 +128,13 @@ def extract_json_data(request):
     if request.body:
         try:
             body = simplejson.loads(request.body)
-            return body
+            if isinstance(body, dict):
```

Comment:
Valid JSON can be string, array or dict. But what would be the purpose of sending such a body to cornice? Colander validators works on object only, isn't it?